### ISSUE TITLE
Broken link in finger tutorial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4681]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#4681|
|Type|release blocker: regression|
|Created|2010-10-11 20:31:35Z|
|Branch|https://github.com/twisted/twisted/tree/finger-tutorial-4681|

http://twistedmatrix.com/documents/current/core/howto/tutorial/protocol.html#auto2

The first link 'service' points to:
http://twistedmatrix.com/documents/10.1.0/api/twisted.application.service.service.html
... which doesn't work, it should point to:
http://twistedmatrix.com/documents/10.1.0/api/twisted.application.service.Service.html

Attachments:

* [finger-tutorial-4681.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/68a/68a47e2ed57a19ed7150556ffd5828dce3d0899a/fbbf5092d0706a3601f81450fe4fdf8bad8c0904.diff) (1217 bytes) - added by lvh on 2010-10-11 21:56:11Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4681 4681
type__release_blocker__regression release blocker: regression
reporter__lvh lvh
priority__normal normal
milestone__ 
branch__branches_finger_tutorial_4681 branches/finger-tutorial-4681
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1286829095000000 1286829095000000
changetime__1290423356000000 1290423356000000
version__None None
owner__lvh lvh
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_1"></a>|@lvh commented|
|-|-|

diff attached

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Introduced by r30082 in resolution of [#4569](https://github.com/twisted/twisted/issues/4569)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @lvh|
|-|-|

Thanks for catching and fixing this, lvh. I might put the IService in "all the generic IService behavior" in a fixed-width format, but other than that, looks great, please add a NEWS file and merge. (There is one typo elsewhere in the file, verison ==> version, if you want to fix that too)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph commented|
|-|-|

lvh, can you please news-file-and-merge ASAP?

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh commented|
|-|-|

(In [30322]) Branching to 'finger-tutorial-4681'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_7"></a>|@lvh set status to `closed`|
|-|-|

(In [30327]) Merge finger-tutorial-4681: fix broken link and typo in finger tutorial.

Author: lvh
Reviewers: jesstess
Fixes: [#4681](https://github.com/twisted/twisted/issues/4681)

The finger tutorial had a broken link to the Service class. Additionally, a minor typo ("verison" instead of "version") was fixed.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_8"></a>|@lvh commented|
|-|-|

Sorry about the delay. svn+ssh was making my life unusually painful and I still don't know why.
